canonicalwebteam/discourse_docs/__init__.py

```diff
@@ -0,0 +1 @@
+from canonicalwebteam.discourse_docs.app import DiscourseDocs  # noqa
```

Comment:
Could we make this:

```suggestion
from canonicalwebteam.discourse_docs.app import DiscourseDocs  # noqa
from canonicalwebteam.discourse_docs.models import DiscourseAPI  # noqa
```

So that, in apps, we can do:

``` python3
from canonicalwebteam.discourse_docs import DiscourseDocs, DiscourseAPI
```